Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Date/index.md

```diff
@@ -1,28 +1,36 @@
 ---
 versionFrom: 7.0.0
+versionTo: 9.0.0
 ---
 
 # Date
 
-The date picker uses a front-end library called [Pikaday](https://github.com/dbushell/Pikaday) to display a UI to pick dates from.
+The date picker uses a front-end library called [Pikaday](https://github.com/dbushell/Pikaday) to display a UI to pick dates.
 
 ![Date picker on frontend](images/date-picker.png)
 
-As of Umbraco Forms 4.4.0 we have added the support for the Pikaday date picker to be localised based on the page the form is rendered on.
+As of Umbraco Forms 4.4.0, Pikaday date picker can be localised based on the page the Form is rendered on.
 
-This displays the picked date in the correct locale. Using JavaScript we update a hidden field with a standard date format to send to the server for storing the record submission in a standard format. This is to avoid locale mixing up dates.
+The date picker displays the picked date in the required locale. Using JavaScript, a hidden field is updated with a standard date format to send to the server for storing the record submission in a standard format. This avoids the locale mixing up the dates.
 
-To achieve this, a Razor partial view is included - you can find it here: `/Views/Partials/Forms/DatePicker.cshtml`.
+To achieve localized date, a Razor partial view is included at `/Views/Partials/Forms/Themes/default/DatePicker.cshtml`.
 
-This includes the Moment.js library to help with the date locale formatting & the appropriate changes to Pikaday to support the locales. If you wish to use a different DatePicker component this is the file that you would customize to your needs.
+The **DatePicker.cshtml** includes the `moment-with-locales.min.js` library to help with the date locale formatting and the appropriate changes to Pikaday to support the locales. If you wish to use a different DatePicker component, edit the **DatePicker.cshtml** file as per your needs.
 
 ## Configure the Year range
+
 The Date picker has a configuration setting to control the number of years shown in the picker. The default value is 10 years.
 
-You can find the settings for changing this configuration in `~/App_Data/UmbracoForms/umbracoforms.config`:
```

Comment:
If this article is still valid for V8 as well as V9, then we should still say that they can configure it from the UmbracoForm.config :) 

So you could add the sentence back and add: _You can find the settings for changing this configuration for Umbraco 8 in `~/App_Data/UmbracoForms/umbracoforms.config`:_

And then also re-add the configuration for V8 as well.

I hope it makes sense 😊

Reply:
It definitely makes sense. Not sure what was I thinking while taking it off.. 😐 Added the content back in 😊